tests/test_get_cve.py

```diff
@@ -0,0 +1,91 @@
+import pytest
+import subprocess
+from subprocess import run
+import tempfile
+import lib.get_cve
+from lib.get_cve import git_log_titles, find_log_titles
+from pathlib import Path
+
+mock_list_titles_1 = [
+    "Add memory requirements",
+    "fix(XX-3444): test1",
+    "fix(XX-3445): test2",
+    "feat(XX-3446): test3",
+]
+
+mock_list_titles_2 = [
+    "Add memory requirements",
+    "fix(CVE-3444): test1",
+    "fix(CVE-3445): test2",
+    "feat(CVE-3446): test3",
+]
+
+mock_result_good = [
+    "fix(CVE-3444): test1",
+    "fix(CVE-3445): test2"
+]
+mock_reponse_data_empty = []
+
+
+class MockCompletedProcess:
+    def __init__(self, returncode, stdout, stderr=""):
+        self.returncode = returncode
+        self.stdout = stdout
+        self.stderr = stderr
+
+
+def mock_empty_log_titles():
+    return []
+
+
+def test_git_log_empty_list(monkeypatch):
+    monkeypatch.setattr(lib.get_cve, 'find_log_titles', mock_empty_log_titles)
+    result = find_log_titles(mock_list_titles_1)
+    assert result == []
+
+
+def mock_success_log_titles():
+    return mock_result_good
+
+
+def test_git_log_success_list(monkeypatch):
+    monkeypatch.setattr(lib.get_cve, 'find_log_titles', mock_success_log_titles)
+    result = find_log_titles(mock_list_titles_2)
+    assert result == mock_result_good
+
+
+@pytest.fixture(scope="module")
+def git_config():
+    git_folder = tempfile.mkdtemp()
+    (Path(git_folder) / "README.md").write_text("# Hello")
+    run(["git", "init", git_folder, "--initial-branch", "main"], check=True)
+    run(["git", "-C", git_folder, "config", "user.email", "you@example.com"], check=True)
+    run(["git", "-C", git_folder, "config", "user.name", "You @ Example"], check=True)
+    run(["git", "-C", git_folder, "add", "."], check=True)
+    run(["git", "-C", git_folder, "commit", "-m", "Initial commit"], check=True)
+
+    return git_folder
+
+
+def test_git_log_success(monkeypatch, git_config):
```

Comment:
The fixture `git_config` creates a real git repository so you can run real git commands on it. Since you are patching the `subprocess.run` execution and result you don't need the fixture.

I think that applies to the other tests that also patch `subprocess.run`